cmd/metrics-v3-cluster-config.go

```
@@ -0,0 +1,49 @@
+// Copyright (c) 2015-2024 MinIO, Inc.
+//
+// This file is part of MinIO Object Storage stack
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package cmd
+
+import "context"
+
+const (
+	configWriteQuorum    = "write_quorum"
+	configRRSParity      = "rrs_parity"
+	configStandardParity = "standard_parity"
+)
+
+var (
+	configWriteQuorumMD = NewGaugeMD(configWriteQuorum,
+		"Maximum write quorum across all pools and sets")
+	configRRSParityMD = NewGaugeMD(configRRSParity,
+		"Reduced redundancy storage class parity")
+	configStandardParityMD = NewGaugeMD(configStandardParity,
+		"Standard storage class parity")
+)
+
+// loadClusterConfigMetrics - `MetricsLoaderFn` for cluster config
+// such as standard and RRS parity.
+func loadClusterConfigMetrics(ctx context.Context, m MetricValues, c *metricsCache) error {
+	clusterDriveMetrics, _ := c.clusterDriveMetrics.Get()
```

Do we really need to ignore this error ?

---

@vadmeste my understanding is that `Get()` will always return a value (either already cached one or a newly retrieved one), so the error can be ignored here, but I could be wrong. I've taken this code from `metrics-v3-cluster-health.go`

@donatello can you please confirm what's the right way?